Assets/Scenes/SEENewWorld.unity

```diff
@@ -31027,9 +31027,9 @@ MonoBehaviour:
       Data: 
   LODCulling: 0.001
   ConfigurationPath:
-    Root: 0
+    Root: 3
     RelativePath: /reflexion/compiler/Reflexion.cfg
-    AbsolutePath: s/reflexion/compiler/Reflexion.cfgexion/compiler/Reflexion.cfg
+    AbsolutePath: 
```

Comment:
What's the reason this path has been removed?

Reply:
Somehow a broken absolute path got checked in. This restores it to use the relative path from StreamingAssets.